### ISSUE TITLE
CSS Variables

### ISSUE THREAD

**Issue by jordanwillis** [hobbes179/colmar_academy#3]
Nice job taking the extra time to learn about CSS variables and applying them to this project!  As you have hopefully realized, using variables is one way to make your CSS more robust and maintainable going forward.  One thing that I wanted to mention however, is to be mindful of using advanced CSS features such as variables because it might not have support in a broad number of browsers.

For example, CSS variables are not support at all by any version of Internet Explorer, which unfortunately is still being used by a very healthy percentage of internet users.  Also, CSS variables is considered an "experimental feature" for most other browsers.

https://developer.mozilla.org/en-US/docs/Web/CSS/var (see Browser Compatibility Section)

Long story short, its perfectly fine to use newer features such as this, however just make sure you understand what percentage of your web sites intended userbase will be impacted by the implementation choice.

Another option that you could use it to explore SASS or LESS, which require pre-processors to compile down some cools features into straight CSS.
